tinyman/v1/staking/__init__.py

```diff
@@ -227,3 +231,80 @@ def prepare_reward_metadata_for_payment(distribution_date: str, cycles_rewards:
         "rewards": [[str(cycle), str(amount)] for (cycle, amount) in cycles_rewards],
     }
     return data
+
+
+def parse_payment_transaction(txn):
```

Comment:
We can add `reward` to function name too.
`parse_reward_payment_transaction`

Reply:
Yeah agreed `parse_reward_payment_transaction` is better